tasker/validation/task_validator.py

```
@@ -1080,6 +1094,111 @@ def validate_parallel_task(self, task, task_id, line_number, parallel_tasks):
+                ipv4_like_pattern = re.compile(r'^(\d+\.){3}\d+$')
+
+                invalid_hostnames = []
+                for hostname in hostnames:
+                    # Check if it looks like an IPv4 address (4 dot-separated numbers)
+                    if ipv4_like_pattern.match(hostname):
+                        # Looks like IPv4, validate strictly as IPv4
+                        try:
+                            ipaddress.IPv4Address(hostname)
+                            # Valid IPv4, no further checks needed
+                            continue
+                        except (ipaddress.AddressValueError, ValueError) as e:
```

Ruff (0.14.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)